app/init/websocket_manager.ts

```diff
@@ -156,13 +185,22 @@ class WebsocketManager {
             return;
         }
 
-        if (appState !== 'active') {
-            this.closeAll();
+        this.cancelAllConnections();
+        if (appState === 'background' && !this.isBackgroundTimerRunning) {
```

Comment:
Do we want to handle any other possible appState?

Reply:
no we only want to handle call this when the app is in the background, no other state, this change was made consciously and with purpose